src/utils/get-file-from-path.ts

```diff
@@ -0,0 +1,26 @@
+import R from 'ramda'
+import fs from 'fs'
+import yaml from 'js-yaml'
+
+import { File } from './read-file'
+
+export const readFileToString = (filePath: string): string =>
```

Comment:
This is premature optimisation IMO. You're using this util function in _one_ place and it's just a wrapper for a simple built-in node.js function.

I'd just get rid off it, that way you also save some LOC in your tests

Reply:
Hm I don't agree:

- it's function composition.
- it hides the detail of `readFile` eventually returning a Buffer.
- it's explicit about the result. It's a string, not a json/buffer/yaml/`File`-type/etc.
- it's easy to mock. I agree, the tests for this function is a bit dumb but just there for completeness.

Reply:
My main counter-argument is that this one-time-only util function costs 20 LOC in tests. To your points:

- it's the most minimal example of function composition i can think off 😛
- it's a simple `.toString()`, IMO this is very clear that it's not a buffer
- `.toString()` is very clear about what it returns already
- ok, fair point
- it's not that much harder to mock `fs.readFileSync` IMO, but ok, small point

I realise this is a nit contest, so I'll resolve this. My main point is: Sometimes the cost of creating abstractions outweighs its benefits e.g. by generating a lot more LOC to maintain.